### DISCUSSION TITLE
Can keepassDX autofill search the "title" field for matches rather than just the URL field

### DISCUSSION THREAD

**Discussion by cjk28** [Kunzisoft/KeePassDX#830] [Ideas]
As per title, my database is such that the url is n the "Title" field, and the URL field is generally empty. With Keepass windows autofill function and Keepass2Android autofill function, this is sufficient for the autofill to find and use the entry. it seems keepassDX is only using the URL field

**Comment by J-Jamet** [comment]
Indeed, the information search function is configured not to search in the title field.
https://github.com/Kunzisoft/KeePassDX/blob/ec63d7534958e7892852860b6bf0f7ca8a9a42a4/app/src/main/java/com/kunzisoft/keepass/database/element/Database.kt#L429

This behavior can be changed but it has been implemented to precisely save the search elements in the right fields. Normally, if you select the correct entry manually once and the settings are set for saving, the field is filled automatically in the corresponding entry and no longer requires selection in the next autofill search.

**Reply by cjk28** [reply]
Thanks for the reply! It indeed does work as you say, but this is not ideal for users such as myself with hundreds of entries wanting to switch from keepass2android. I did try editing my database to transpose the URL from your field to URL field but this will not be a walk in the park as the CSV export strips some info. Would you consider changing that Boolean to true or exposing it in the UI?

**Reply by cjk28** [reply]
in fact, if i update my database by moving th URL from the Title field to the URL field, the auto-type feature of keepass windows application no longer works. Strange that the desktop client works this way, surely it makes no sense to have the URL duplicated in two fields. Would you considere enabling serachInTtiels for keepassDX?

**Reply by J-Jamet** [reply]
I just changed the setting, however, it will need to be tested to see if there is any side effect.

**Comment by cjk28** [comment]
Awesome! If you can publish to the beta channel I can try and report back, will create some new entries with URL in the URL field etc to try and break itOn 12 Jan 2021 08:29, Jérémy JAMET <notifications@github.com> wrote:
I just changed the setting, however, it will need to be tested to see if there is any side effect.

**Comment by cjk28** [comment]
Hi, it's working nicely except it will crash every time clicking on a relevant field in a browser e.g. chrome unless my database is open/unlocked. Android 10, e.g. site: GitHub

**Comment by J-Jamet** [comment]
The problem comes from https://github.com/Kunzisoft/KeePassDX/issues/852, I will create a new version.

**Reply by cjk28** [reply]
Working great, thanks a lot!
